plugins/mcreator-core/procedures/entity_spawn_z.json

```diff
@@ -0,0 +1,26 @@
+{
+  "args0": [
+    {
+      "type": "input_value",
+      "name": "entity",
+      "check": "Entity"
+    }
+  ],
+  "output": "Number",
+  "colour": "%{BKY_MATH_HUE}",
+  "mcreator": {
+    "toolbox_id": "entitydata",

```

Comment:
I think this procedure only applies to players as other entities do not really have a (re)spawn point

Reply:
:heavy_check_mark: Fixed